### PR TITLE
Fix #859 : Added descriptive Beneficiary Spinner TPT

### LINKED ISSUE #859: Beneficiary Spinner should be more descriptive
**Summary:**

Beneficiary Spinner should be more descriptive

**Steps to reproduce:**

Make TPT Transfer

**Expected behaviour:**

Atleast, Beneficiary Name along with Account Number should be displayed

**Observed behaviour:**

Instead it's not

**Device and Android version:**

Xiaomi, Marshmallow

**Screenshots:**

screenshot_2018-06-15-16-20-00-871_org mifos mobilebanking

### PR DESCRIPTION
**Pull request by miPlodder**
Fixes #859 

pr_851

pr_859

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/res/layout/beneficiary_spinner_layout.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (therajanmaurya) [inline app/src/main/res/layout/beneficiary_spinner_layout.xml:1]: this xml file name should be descriptive_spinner_layout.xml

File: app/src/main/java/org/mifos/mobilebanking/ui/fragments/ThirdPartyTransferFragment.java [hunk 2]
```diff
@@ -190,8 +191,8 @@ public void showUserInterface() {
         spPayFrom.setAdapter(payFromAdapter);
         spPayFrom.setOnItemSelectedListener(this);
 
-        beneficiaryAdapter = new ArrayAdapter<>(getActivity(), android.R.layout.simple_spinner_item,
-                listBeneficiary);
+        beneficiaryAdapter = new BeneficiarySpinnerAdapter(getActivity(),
+                R.layout.beneficiary_spinner_layout, listBeneficiary);
```
Comment (therajanmaurya) [inline app/src/main/java/org/mifos/mobilebanking/ui/fragments/ThirdPartyTransferFragment.java:195]: Spinner Title should be `Select Beneficiary`

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/beneficiary_spinner_layout.xml (hunk 1 above)
Comment (miPlodder) [reply app/src/main/res/layout/beneficiary_spinner_layout.xml:1]: @therajanmaurya  I used a similar naming convension as [File](https://github.com/openMF/mifos-mobile/blob/development/app/src/main/res/layout/account_spinner_layout.xml). Should I rename both files to `descriptive_beneficiary_spinner_layout.xml` and `descriptive_account_spinner_layout.xml`

File: app/src/main/java/org/mifos/mobilebanking/ui/fragments/ThirdPartyTransferFragment.java (hunk 2 above)
Comment (miPlodder) [reply app/src/main/java/org/mifos/mobilebanking/ui/fragments/ThirdPartyTransferFragment.java:195]: @therajanmaurya I have made this change and update the image for this PR.

**Review by therajanmaurya — APPROVED** [review]

**Pull request merged**